pwiz/data/vendor_readers/Waters/SpectrumList_Waters.cpp

```diff
@@ -135,9 +135,12 @@ PWIZ_API_DECL SpectrumPtr SpectrumList_Waters::spectrum(size_t index, DetailLeve
     int msLevel;
     CVID spectrumType;
     translateFunctionType(WatersToPwizFunctionType(rawdata_->Info.GetFunctionType(ie.function)), msLevel, spectrumType);
+    

```

Comment:
What does this have to do with the file extensions?

Reply:
It doesn't. It fixes a crash a user got converting their data with a PDA function. It makes no difference to MS data. Any particular reason I need to separate it out? It would have crashed Skyline too if it tried to import this data.

Reply:
Okay. Fine. Looks like it is 2 separate commits. Just don't squash when you merge. Thanks for clarifying.